guides/common/assembly_managing-application-life-cycles.adoc

```diff
@@ -1,7 +1,4 @@
-[id="Creating_an_Application_Life_Cycle_{context}"]
-= Managing Application Life Cycles
-
-This chapter outlines the application life cycle in {Project} and how to create and remove application life cycles for {Project} and {SmartProxy}.
+include::modules/con_managing-application-life-cycles.adoc[]
```

Comment:
Mark -- see my comment for `assembly_synchronizing-content-between-foreman.adoc`